packages/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -411,23 +449,21 @@ class DescopeWc extends BaseDescopeWc {
       const defaultMessage = sdkResp?.response?.url;
       const defaultDescription = `${sdkResp?.response?.status} - ${sdkResp?.response?.statusText}`;
 
-      this.logger.error(
+      this.logger().error(
         sdkResp?.error?.errorDescription || defaultMessage,
         sdkResp?.error?.errorMessage || defaultDescription
       );
       return;
     }
 
     const errorText = sdkResp.data?.screen?.state?.errorText;
-    if (errorText) {
-      this.logger.error(errorText);
-    }
-
     if (sdkResp.data?.error) {
-      this.logger.error(
+      this.logger().error(
         `[${sdkResp.data.error.code}]: ${sdkResp.data.error.description}`,
-        sdkResp.data.error.message
+        `${errorText ? `${errorText} - ` : ''}${sdkResp.data.error.message}`
```

Comment:
Why do we need to "build" the error string here and we cannot use the existing pattern?

Reply:
the screen error is a good link between the actual error and the user error. in order to connect between them we concat them instead of producing 2 errors that mean the same.